geopandas/array.py

```diff
@@ -1208,30 +1208,6 @@ def take(self, indices, allow_fill=False, fill_value=None):
             result[~shapely.is_valid_input(result)] = None
         return GeometryArray(result, crs=self.crs)
 
-    def _fill(self, idx, value):
```

Comment:
I merged this helper function in `fillna` itself, because that was the only place where it was actually used